subsys/bluetooth/common/Kconfig

```diff
@@ -49,7 +49,9 @@ config BT_BUF_ACL_TX_COUNT
 	  command response.
 
 config BT_BUF_ACL_RX_SIZE
-	int "Maximum supported ACL size for incoming data"
+	int "Maximum supported ACL size for incoming data" if BT_CLASSIC ||\
+							      !BT_CTLR || \
+							      BT_CTLR_DATA_LEN_UPDATE_SUPPORT
```

Comment:
Can you elaborate in the PR body or commit message why BT_BUF_ACL_RX_SIZE is changed to be promptless for builds including a controller? I would assume a controller may recombine multiple PDUs into a single HCI ACL Data. This regardless of data length update support

Reply:
my fault... the comments are wrong then? The value is used for final recombined *data packet* and not *HCI packet* buffers, right?

```
include/zephyr/bluetooth/buf.h-73-
include/zephyr/bluetooth/buf.h-74-/** Data size needed for HCI ACL RX buffers */
include/zephyr/bluetooth/buf.h:75:#define BT_BUF_ACL_RX_SIZE BT_BUF_ACL_SIZE(CONFIG_BT_BUF_ACL_RX_SIZE)
include/zephyr/bluetooth/buf.h-76-
include/zephyr/bluetooth/buf.h-77-/** Data size needed for HCI Event RX buffers */
--
include/zephyr/bluetooth/buf.h-87-
include/zephyr/bluetooth/buf.h-88-/** Data size needed for HCI ACL, HCI ISO or Event RX buffers */
include/zephyr/bluetooth/buf.h:89:#define BT_BUF_RX_SIZE (MAX(MAX(BT_BUF_ACL_RX_SIZE, BT_BUF_EVT_RX_SIZE), \
include/zephyr/bluetooth/buf.h-90-                          BT_BUF_ISO_RX_SIZE))
include/zephyr/bluetooth/buf.h-91-
```

And we can have asymmetric MTU at L2CAP level (different Tx an Rx MTU)?  User Kconfig configurable Tx MTU and derive Rx MTU?

```
include/zephyr/bluetooth/l2cap.h-37-
include/zephyr/bluetooth/l2cap.h-38-/** Maximum Transmission Unit (MTU) for an incoming L2CAP PDU. */
include/zephyr/bluetooth/l2cap.h:39:#define BT_L2CAP_RX_MTU (CONFIG_BT_BUF_ACL_RX_SIZE - BT_L2CAP_HDR_SIZE)
include/zephyr/bluetooth/l2cap.h-40-
```

Reply:
Yes, I believe you are right. It looks like the comments are wrong